### ISSUE TITLE
 Suggestion: More intuitive settings pages

### ISSUE THREAD

**Issue by Ace2213** [humhub/humhub#3694]
The Account Settings > Settings page doesn't really need to exist. It doesn't serve a purpose that other pages already don't serve. 

![image](https://user-images.githubusercontent.com/12226710/67145814-fa30ee80-f25a-11e9-8bc5-ab843d5ccaa1.png)

Everything in that page can be moved to the Profile page and make more sense there.

Tags make more sense in Profile as that's where they appear. And I don't think they serve any purpose other than visual profile information, just like the General page under Profile. 

The rest of the options can be moved as they are. The entire "Basic Settings" and "Connected Accounts" tabs can be moved under profile.

**Comment by ArchBlood** [comment]
I'd agree somewhat, but I'd also think that if they were added it would look a little over-bloated.

**Comment by Ace2213** [comment]
Then perhaps leave that page for account related settings and leave all profile related settings under Profile?

Account Settings can have two tabs, Basic Settings and Credentials (or a more user-friendly name like Login Information)

Basic Settings can hold things like Language and Timezone. Credentials can have Change Email, Change Password and Connected Accounts, all in the same tab.

Change Username can also be folded into General.

**Comment by ArchBlood** [comment]
I myself think both Change Username and Change Email should be under a sub-tab similar to how profile fields are setup, would clean up the account setting tabs.

**Comment by Ace2213** [comment]
Agreed. They can both be in Profile > General. That way Connected Accounts and Change Password can be unified under "Login options"

**Comment by Ace2213** [comment]
It wasn't as hard as I thought it would be. It required modifying [AccountSettingsMenu.php](https://github.com/humhub/humhub/blob/master/protected/humhub/modules/user/widgets/AccountSettingsMenu.php) (to add the tabs) and [AccountProfilMenu.php](https://github.com/humhub/humhub/blob/master/protected/humhub/modules/user/widgets/AccountProfilMenu.php) (to hide the tabs) and then modifying each view individually to make the tabs open under the right page.

But it was worth it. The options make a lot more sense now. I'd still like it if change Email, change Password and Connected Accounts were all added under one tab, but for now this is fine and it's probably a lot more risky modifying the tabs themselves.

![image](https://user-images.githubusercontent.com/12226710/69500977-10c40880-0edf-11ea-9efd-32b6aa41f7c9.png)

![image](https://user-images.githubusercontent.com/12226710/69500993-3e10b680-0edf-11ea-9dd4-46943ab7b66e.png)

In these views I changed [this line](https://github.com/humhub/humhub/blob/master/protected/humhub/modules/user/views/account/changeEmail.php#L7) from `_userProfileLayout.php` to `_userSettingsLayout.php`
![image](https://user-images.githubusercontent.com/12226710/69501086-25ed6700-0ee0-11ea-9f4c-b9c3c409dce9.png)

**Comment by Ace2213** [comment]
I took it a step further. I turned Account Settings into Credentials and balanced out the number of options between them.

![image](https://user-images.githubusercontent.com/12226710/69501802-2f2e0200-0ee7-11ea-99d2-61a38f5279b3.png)


![image](https://user-images.githubusercontent.com/12226710/69501795-1c1b3200-0ee7-11ea-8167-f66f4ee73404.png)
